src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    
+    <div class="frame">
```

Comment:
Don't add empty lines between parent and child elements
```suggestion
    <div class="frame">
```